src/containers/GraphSettingsContainer.tsx

```diff
@@ -70,6 +75,20 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps> {
         ...this.getGraphParams(),
         injectServiceNodes: this.props.showServiceNodes
       });
+    } else if (this.props.showSecurity && this.props.showSecurity !== prevProps.showSecurity) {
+      // when turning on security we need to perform a fetch, because we don't pull security data by default
+      store.dispatch(
+        // @ts-ignore
+        GraphDataActions.fetchGraphData(
+          this.props.namespace,
+          this.props.graphDuration,
+          this.props.graphType,
+          this.props.injectServiceNodes,
+          this.props.edgeLabelMode,
+          this.props.showSecurity,
+          this.props.node
+        )
+      );
```

Comment:
How does GraphFilterActions get all of the properties used when calling GraphDataActions.fetchGraphData?

Reply:
What do you mean? GraphFilterActions doesn't look involved.

But let me guess and, possibly, answer... Instead of using `bindActionCreators` you would need to explicitly declare a function.  Code would look like:
```
toggleGraphSecurity: (...props) => {
  dispatch(GraphFilterActions.toggleGraphSecurity());
  dispatch(GraphDataActions.fetchGraphData(...arguments));
}
```
The redux "more correct" way involves too much parameter passing. I really find that calling dispatch directly is cleaner. So, no problem.

Reply:
OK, I see what you are saying.  I guess I am not sure which approach I prefer.  The redux way puts logic in one place, so that is good.  Although the data fetch become less obvious and more of a side-effect.  Both approaches have benefits.